src/sickle.ts

```diff
@@ -36,6 +36,11 @@ function classHasSuperClass(classNode: ts.ClassDeclaration): boolean {
   return false;
 }
 
+// trim removes leading and trailing whitespace from a string.
+function trim(str: string): string {
```

Comment:
Looks like ES5 has trim already, and I just checked that node v0.12.0 has it, so might as well use that.